Doc/c-api/tuple.rst

```diff
@@ -35,6 +35,10 @@ Tuple Objects
 
    Return a new tuple object of size *len*, or *NULL* on failure.
 
+   .. versionchanged:: 3.7
+
+      ``PyTuple_New(0)`` now does not raise exceptions.
```

Comment:
This is very imprecise. Why does it not raise exceptions? Because it always succeeds?